packages/components/src/components/teaser-media/teaser-media.stories.ts

```diff
@@ -6,254 +6,214 @@ const { argTypes, parameters } = storybookDefaults('sd-teaser-media');
+        </div>
+      </sd-teaser-media>
+    </div>
+
+    <div style="margin-bottom: 40px; width: 600px">
+      <sd-teaser-media variant="primary-100">
+        <img slot="media" src="./placeholders/images/architecture.jpg" class="aspect-video object-cover" alt="Test" />
+        <h3 slot="headline">Headline Media Teaser (primary-100)</h3>
+        <div class="flex flex-col gap-4">
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </div>
+      </sd-teaser-media>
+    </div>
+
+    <div style="margin-bottom: 40px; width: 600px">
+      <sd-teaser-media variant="neutral-100">
+        <img slot="media" src="./placeholders/images/architecture.jpg" class="aspect-video object-cover" alt="Test" />
+        <h3 slot="headline">Headline Media Teaser (neutral-100)</h3>
+        <div class="flex flex-col gap-4">
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </div>
+      </sd-teaser-media>
+    </div>
+
+    <div style="margin-bottom: 40px; width: 600px">
+      <sd-teaser-media variant="gradient-dark">
+        <img slot="media" src="./placeholders/images/architecture.jpg" class="aspect-video object-cover" alt="Test" />
+        <h3 slot="headline">Headline Media Teaser (gradient-dark)</h3>
+        <div class="flex flex-col gap-4">
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </div>
+      </sd-teaser-media>
+    </div>
+
+    <div style="margin-bottom: 40px; width: 600px">
+      <sd-teaser-media variant="gradient-white">
+        <img slot="media" src="./placeholders/images/architecture.jpg" class="aspect-video object-cover" alt="Test" />
+        <h3 slot="headline">Headline Media Teaser (gradient-white)</h3>
+        <div class="flex flex-col gap-4">
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </div>
+      </sd-teaser-media>
+    </div>
+  `
 };
 
 /**
- * Use the 'default', 'media', 'meta', 'expandable' and 'headline' slots to add content to the teaser. Please make sure to use semantically correct headline tags for the `headline` slot to provide accessible content.
+ * - Use the `default` slot to display main information and/or action elements below the headline.
+ * - Use the `media` slot to add an image to the teaser.
+ * - Use the `headline` slot to display titles. It should always contain a <h*> element.
  */
-export const Slots = {
-  parameters: {
-    controls: { exclude: ['default', 'media', 'meta', 'headline', 'expandable'] }
-  },
-  render: (args: any) => {
-    return html`
-      ${['default', 'media', 'meta', 'headline', 'expandable'].map(slot => {
-        let value = `<div slot='${slot}' class="slot slot--border slot--background h-16"></div>`;
+export const DefaultMediaAndHeadlineSlot = {
+  name: 'Default, Media and Headline Slot',
+  render: () => html`
+    <div style="margin-bottom: 40px; width: 600px">
```

Comment:
Do we need to specify the teaser width for any specific reason? Also, if it's only rendering one teaser, there's no need to have a margin

Reply:
To prevent them to be to big. Just a better user experience